egs/wsj/s5/steps/nnet3/make_cnn_tdnn_configs.py

```diff
@@ -0,0 +1,397 @@
+#!/usr/bin/env python
+
+# we're using python 3.x style print but want it to work in python 2.x,
+from __future__ import print_function
+import re, os, argparse, sys, math, warnings
+import numpy as np
```

Comment:
whenever I use numpy I add a check on the top where the script dies with an Exception which says that Kaldi recipes by default do not assume numpy, but this recipe requires numpy.